avro-builder/tests/codegen-14/src/main/avro/vs14/MoneyRange.avsc

```diff
@@ -0,0 +1,47 @@
+{
+  "type": "record",
+  "name": "MoneyRange",
+  "namespace": "vs14",
+  "doc": "A value of money",
+  "fields": [
+    {
+      "type" : {
+        "type" : "record",
+        "name" : "Amount",
+        "namespace" : "vs14",
+        "doc" : "Represents an amount of money",
+        "fields" : [ {
+          "name" : "currencyCode",
+          "type" : "string",
+          "doc" : "Currency code v$",
+          "compliance" : "NONE"
+        }, {
+          "name" : "amount",
+          "type" : "string",
+          "doc" : "The amount of money as a real number string, See https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html#BigDecimal-java.lang.String-",
+          "compliance" : {
```

Comment:
i think this schema is too large for what its testing. can you trim it down to just the features youre testing for?

Reply:
This is being used to test reuse of internal record type. Updated method name to reflect that